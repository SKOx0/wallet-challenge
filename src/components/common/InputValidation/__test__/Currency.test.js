import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import CurrencyValidation from '../Currency';

describe('<CurrencyValidation />', () => {
  it('deve ser igual ao snapshot', () => {
    const renderedComponent = shallow(<CurrencyValidation />);

    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
