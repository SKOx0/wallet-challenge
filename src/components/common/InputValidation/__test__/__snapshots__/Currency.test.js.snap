// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrencyValidation /> deve ser igual ao snapshot 1`] = `
<Field
  component={[Function]}
/>
`;
